content/v2.0/reference/cli/influx/secret/delete.md

```diff
@@ -0,0 +1,28 @@
+---
+title: influx secret delete
+description: The 'influx secret delete' command deletes secrets.
+menu:
+  v2_0_ref:
+    name: influx secret delete
+    parent: influx secret
+weight: 101
+v2.0/tags: [secrets]
+---
+
+The `influx secret delete` command deletes secrets.
+
+## Usage
+```
+influx secret delete [flags]
+influx secret delete [command]
```

Comment:
think we can drop "influx secret delete [command] here? 
then use the same term "flags" or "options" across the board?

Reply:
See previous comment

Reply:
If delete is the only command, think we can drop [command]?
...reads like it's a command with other sub-commands available, like `secret` 
think in other places we say?:
## Usage
influx secret delete
influx secret delete [flags]

Reply:
You're right. I noticed the context and made this update this morning.